prod-aws/kafka-shared-msk/TERRAFORM_LOCALLY.md

```diff
@@ -0,0 +1,35 @@
+## How to work with Terraform locally
+This is meant to be used only in **unexpected** situations like fixing the state or debugging the kafka provider by the pubsub admins
```

Comment:
Realised these instructions don't work for MSK clusters. Would be better to just replace with TBD